client/DeckSummary.jsx

```diff
@@ -49,7 +49,8 @@ class DeckSummary extends React.Component {
         let combinedCards = _.union(this.props.deck.plotCards, this.props.deck.drawCards);
 
         _.each(combinedCards, (card) => {
-            let type = card.card.type_name;
+            let typeCode = card.card.type;
+            let type = typeCode[0].toUpperCase() + typeCode.slice(1);
```

Comment:
I hate javascript sometimes, what does this do?

Reply:
Capitalizes the first letter of the type. "character" => "Character", "location" => "Location", etc. I could introduce a method (either extend the prototype of String, or add a specific method for converting type to its name) but it seemed overkill for just 2 places.